apps/mobile/global.css

```
@@ -2,55 +2,65 @@
 @theme {
-	--radius: 0.5rem;
+	--radius: 0.625rem;
+	--font-sans:
+		"Geist", -apple-system, BlinkMacSystemFont, "Segoe UI", system-ui,
+		sans-serif;
+	--font-mono: "Geist Mono", ui-monospace, "SF Mono", monospace;
 }
 
 @layer theme {
 	:root {
 		@variant light {
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)